modules/SitecoreImageBuilder/1.0.0/Public/Invoke-PackageRestore.ps1

```diff
@@ -39,13 +39,11 @@ function Invoke-PackageRestore
     $ProgressPreference = "SilentlyContinue"
 
     $sitecoreDownloadUrl = "https://dev.sitecore.net"
-
-    # Load packages file
-    $packagesFile = Get-Item -Path (Join-Path $PSScriptRoot "..\..\..\..\sitecore-packages.json")
-    $packages = $packagesFile | Get-Content | ConvertFrom-Json
-
     $destinationPath = $Destination.TrimEnd('\')
 
+    # Load packages
+    $packages = $packages = Get-Packages
```

Comment:
not sure I get this.. why is there a double $packages = ?

Reply:
good spot and it's gone now :) too much copy paste ;)